src/com/carolinarollergirls/scoreboard/jetty/JettyServletScoreBoardController.java

```diff
@@ -29,14 +29,13 @@
 import org.eclipse.jetty.servlet.ServletHolder;
 
 import com.carolinarollergirls.scoreboard.ScoreBoardManager;
+import com.carolinarollergirls.scoreboard.core.ScoreBoard;
 import com.carolinarollergirls.scoreboard.json.JSONStateManager;
-import com.carolinarollergirls.scoreboard.model.ScoreBoardModel;
-// import org.eclipse.jetty.util.resource.Resource;
 
 
 public class JettyServletScoreBoardController {
-    public JettyServletScoreBoardController(ScoreBoardModel model, JSONStateManager jsm) {
-        scoreBoardModel = model;
+    public JettyServletScoreBoardController(ScoreBoard model, JSONStateManager jsm) {
```

Comment:
We should probably change args like this to `sb`

Reply:
I've eliminated all occurences of the string 'model' from the java code now.